cloudbuild.yaml

```diff
@@ -19,7 +19,50 @@ steps:
       echo "Write .env.cloud_build to disk, to provide consistent project configuration (no secrets) in subsequent steps"
       deploy/make_cloud_build_env_file.sh "${BRANCH_NAME}" "${COMMIT_SHA}"
 
-  - id: "Build image"
+  - id: "Build server and postgres images for testing"
+    name: "gcr.io/cloud-builders/docker"
+    script: |
+      #!/usr/bin/env bash
+      set -o errexit
+      set -o pipefail
+      set -o nounset
+      
+      echo "Get cloud build project configuration values"
+      source .env.cloud_build
+
+      echo "Build new server and postgres db images for testing"
+      docker compose --env-file .env.cloud_build up --build -d 
+
+  - id: "Run tests and produce coverage report"
+    name: 'gcr.io/cloud-builders/docker'
+    script: |
+      #!/usr/bin/env bash
+      set -o errexit
+      set -o pipefail
+      set -o nounset
+
+      echo "Get cloud build project configuration values"
+      source .env.cloud_build
+
+      echo "Run tests and create coverage report"
+      docker compose --env-file .env.cloud_build exec -T server coverage run -m pytest
+      docker compose --env-file .env.cloud_build exec -T server coverage report --show-missing
```

Comment:
Probably worth adding a TODO comment here noting that uploading the coverage report somewhere is a next-step, and that for now coverage reports need to be dug up in the Cloud Build logs. Would be a helpful breadcrumb for anyone looking for the coverage reports in the meantime, and a good reminder (in addition to the issue about it).

Reply:
Issue #75  & [Added a TODO to upload coverage report as a next step and make report…](https://github.com/PHACDataHub/cpho-phase2/commit/ec97c7e6505b59165bb5ae66e8f9a7ce8f4af5ce)